src/sass/blocks/_about.scss

```
@@ -0,0 +1,38 @@
+.about {
+	min-height: 985px;
+	background-color: #f8fafa;
+	padding-top: 130px;
+
+	&__wrapper {
+		display: flex;
+		justify-content: space-around;
+		padding-top: 130px;
```

you can use variable for this value

---

Thanks. I added mixins.
```suggestion
		@include flex_space_around;
```